Jint/Engine.cs

```diff
@@ -59,32 +80,35 @@ public sealed class Engine
         internal readonly ArgumentsInstancePool _argumentsInstancePool;
         internal readonly JsValueArrayPool _jsValueArrayPool;
 
-        public ITypeConverter ClrTypeConverter;
+        public readonly ITypeConverter ClrTypeConverter;
```

Comment:
Why was this done? It is now no longer possible to override with a custom `ITypeConverter`

Reply:
Just an oversight. Maybe submit a PR with a test case that protects from happening again?

Reply:
See #654  for pr